pipeline/Dockerfile

```diff
@@ -12,7 +12,7 @@ RUN  apt-get update && \
      rm -rf /var/lib/apt/lists/*
 
 # Kubectl
-ENV KUBECTL_VERSION "v1.18.13"
+ENV KUBECTL_VERSION "v1.19.9"
```

Comment:
It should be fine to use a newer version of kubectl than that of the kube-apiserver, right?
Until we have the kubepray pipeline in place we run 1.18 i think.

Reply:
Yes that should be fine I think